app/src/test/java/org/mozilla/fenix/library/bookmarks/BookmarkDeselectNavigationListenerTest.kt

```diff
@@ -6,6 +6,7 @@ package org.mozilla.fenix.library.bookmarks
 
```

Comment:
Note to reviewer: these changes preserve the original behavior of the tests.  This is often outside of the scope implied by the test name.  I'm happy to change this to test what the name suggests if you would prefer.